src/opera_disp_tms/create_tile_map.py

```diff
@@ -31,7 +31,7 @@ def get_tile_extent(info: dict, output_folder: Path) -> None:
         json.dump(extent, outfile)
 
 
-def create_tile_map(output_folder: str, input_rasters: list[str], scale_range: List[float] = None) -> None:
+def create_tile_map(output_folder: str, input_rasters: list[str], scale_range: Optional[list[float]] = None):
```

Comment:
Starting with Python 3.10, `Optional` can be replaced with `X | None`, so `list[float] | None` in this case.

Reply:
and I think we're doing python >=3.10 for this application, so we're safe to use that syntax